tests/test_job_management.py

```diff
@@ -1162,6 +1168,127 @@ def test_email_very_long_request_url(dispatcher_long_living_fixture, dispatcher_
 This might be fixed in a future release.""" in email_data
 
 
+@pytest.mark.not_safe_parallel
+@pytest.mark.parametrize("use_scws_value", ['form_list', 'user_file', 'no', None])
+@pytest.mark.parametrize("passing_scw_list", [True, False])
+def test_email_scws_list(dispatcher_live_fixture,
+                         dispatcher_local_mail_server,
+                         use_scws_value,
+                         passing_scw_list):
+    DispatcherJobState.remove_scratch_folders()
+
+    server = dispatcher_live_fixture
+    logger.info("constructed server: %s", server)
+
+    # let's generate a valid token
+    token_payload = {
+        **default_token_payload,
+        "roles": "unige-hpc-full, general",
+    }
+    encoded_token = jwt.encode(token_payload, secret_key, algorithm='HS256')
+
+    params = {
+        'query_status': "new",
+        'product_type': "dummy",
+        'query_type': "Real",
+        'instrument': 'empty-async',
+        'use_scws': use_scws_value,
+        'token': encoded_token
+    }
+
+    if use_scws_value == 'user_file':
+        scw_list_file_obj = None
+        if passing_scw_list:
+            file_path = DispatcherJobState.create_scw_list_file(list_length=5)
+            scw_list_file = open(file_path).read()
+            scw_list_file_obj = {"user_scw_list_file": scw_list_file}
+        # we are supposed to send a file, so it has to be a post request
+        jdata = ask(server,
+                    params,
+                    method='post',
+                    max_time_s=150,
+                    expected_query_status=None,
+                    expected_status_code=None,
+                    files=scw_list_file_obj
+                    )
+    elif use_scws_value == 'no':
+        # no list should be passed
+        jdata = ask(server,
```

Comment:
this does not test the case when `use_scws_value == "no"` but something is passed. Similarly in other cases

Reply:
What is missing in the parameters for the request?

Reply:
I mean when `use_scws_value == "no"` but scw_list is passed.

Reply:
ok, and in that case what behavior do we expect? consider the `scw_list` passed and change the value of `use_scws` to `form_list` so to use the passed list in the frontend?

Reply:
Normally, such a request can not reasonably emerge in frontend or API, and if it is sent, it has to be considered as self-contradictory, ambiguous, and hence return some 400 error.

It might be that it is sent by frontend at some point. But let's make it 400 and leave a comment note